src/pages/youngho/Login/Login.js

```diff
@@ -1,10 +1,103 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      loginId: '',
+      loginPassword: '',
+    };
+  }
+
+  handleLoginInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleLogin = () => {
+    fetch('http://10.58.0.84:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.loginId,
+        password: this.state.loginPassword,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log('결과: ', result));
+  };
+
+  // handleLogin = () => {
+  //   fetch('http://10.58.0.84:8000/users/signup', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       name: '김영호',
+  //       email: this.state.loginId,
+  //       password: this.state.loginPassword,
+  //       phone_number: '01012345678',
+  //       date_of_birth: '1995-07-07',
+  //       gender: 'M',
+  //       address: 'Seoul',
+  //     }),
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => console.log('결과: ', result));
+  // };
+
   render() {
+    const { checkIdPassword, handleLoginInput } = this;
+    const { loginId, loginPassword } = this.state;
+
     return (
       <div>
-        <h3>로그인</h3>
+        <div className="container">
+          <header className="header">Westagram</header>
+          <form className="loginForm" onKeyUp={checkIdPassword}>
+            <input
+              id="id"
+              name="loginId"
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+              onChange={handleLoginInput}
+            />
+            <input
+              id="password"
+              name="loginPassword"
+              type="password"
+              placeholder="비밀번호"
+              onChange={handleLoginInput}
+            />
+            <Link to="/main">
+              <button
+                onClick={this.handleLogin}
+                className={
+                  loginId.length >= 5 &&
+                  loginPassword.length >= 5 &&
+                  loginId.includes('@')
+                    ? 'buttonActivate'
+                    : 'buttonDisabled'
+                }
+                id="loginButton"
+                disabled={
+                  loginId.length >= 5 &&
+                  loginPassword.length >= 5 &&
+                  loginId.includes('@')
+                    ? false
+                    : true
+                }
+              >
+                로그인
+              </button>
```

Comment:
```suggestion
              <button
                onClick={this.handleLogin}
                className={condition ? 'buttonActivate' : 'buttonDisabled'}
                id="loginButton"
                disabled={ condition ? false : true}
              >
                로그인
              </button>
```
해당 조건을 condition이라는 변수에 담아서 사용하시면 가독성이 더 좋아지겠네요!
ex) 
```
const condition = loginId.length >= 5 && loginPassword.length >= 5 && loginId.includes('@');
```

Reply:
참고로 위의 변수는 render 함수 내에서 선언해주시면 됩니다!!

Reply:
반영 완료했습니다!